Exiled.Events/EventArgs/Scp096/StartPryingGateEventArgs.cs

```diff
@@ -56,7 +54,10 @@ public StartPryingGateEventArgs(Scp096Role scp096, Player player, PryableDoor ga
         /// <summary>
         ///     Gets the <see cref="PryableDoor" /> to be pried open.
         /// </summary>
-        public Door Door { get; }
+        public Door Door => Gate;
+
+        /// <inheritdoc cref="Door"/>
```

Comment:
```suggestion
        ///     Gets the <see cref="Gate" /> to be pried open.
```
Docs should match the return type